Wireframe/index.html

```diff
@@ -4,10 +4,40 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <main>
+      <div class="header">
+      <h1>Welcome to Git</h1>
+      <p>Tracking files and directories made easy</p>
+      </div>
+      <div class="article1">
+        <img src="./image1.png" alt="">
+        <h2>What is Git?</h2>
+        <p>Git is a distributed version control system that allows multiple developers to collaborate on projects efficiently. It tracks changes in files and directories, enabling users to revert to previous versions, branch out for new features, and merge updates seamlessly. Git is widely used in software development for its speed, flexibility, and ability to handle large projects.</p>
+        <button>READ MORE</button>  
+      </div>
+      <div class="articleContainer">
+      <div class="article">
+        <img src="./image2.png" alt="">
+        <h2>Why do developers need Git?</h2>
+        <p>Developers need Git for effective version control and collaboration. It allows them to track changes, revert to previous versions, and manage code integrity. With branching capabilities, developers can work on new features without disrupting the main codebase. Git's distributed nature ensures that every developer has a complete project copy, safeguarding against data loss. Additionally, its detailed history of changes facilitates project evolution tracking and contribution reviews. Overall, Git streamlines workflows and enhances productivity in software development.</p>
+        <button>READ MORE</button>  
+      </div>
+      <div class="article">
+        <img src="./image3.jpeg" alt="">
+        <h2>What is a Git branch?</h2>
+        <p>A Git branch is a parallel version of a repository that allows developers to work on different features or fixes independently without affecting the main codebase. Each branch represents an isolated environment where changes can be made, tested, and refined. Once the work on a branch is complete, it can be merged back into the main branch (usually called "main" or "master"), integrating the new changes into the overall project. This branching strategy enhances collaboration and helps manage the development process efficiently.</p>
+        <button>READ MORE</button>  
+      </div>
+      </div>
+      <br>
```

Comment:
using break rules i.e. `<br>` is ok, but if you are using this for design purposes or spacing, it is better and easier to use `CSS`.

Instead of 2 `<br>` here you could add `margin-bottom` on the `<div class="articleContainer">`, but you would need to give that div a height or you can use something like `overflow: hidden`, but it's bit hacky but works. There is also something called flexbox, but I wont go into that now.